packages/autocomplete-core/completion.ts

```diff
@@ -0,0 +1,52 @@
+import { AutocompleteState, RequiredAutocompleteOptions } from './types';
+import {
+  getSuggestionFromHighlightedIndex,
+  getRelativeHighlightedIndex,
+} from './utils';
+
+interface GetCompletionProps<TItem> {
+  state: AutocompleteState<TItem>;
+  props: RequiredAutocompleteOptions<TItem>;
+}
+
+export function getCompletion<TItem>({
+  state,
+  props,
+}: GetCompletionProps<TItem>): string | null {
+  if (!props.showCompletion || !state.isOpen || state.status === 'stalled') {
```

Comment:
Does it mean it won't work without dropdown?

Reply:
What I have in mind for unblocking the use case "autofill only" is to use:

```
{
  shouldDropdownOpen: () => false,
}
```

`state.isOpen` would therefore be `true`, be the dropdown will hide. This could be packaged as a preset.

Do you see any alternatives?

Reply:
Ah okay, so with `shouldDropdownOpen: () => false`, `state.isOpen` still can be `true`?

Reply:
Right now, yes, it's only an indication for the renderer. It might change later though.

Reply:
Two `Open`s sound colliding. What do you think of renaming `shouldDropdownOpen` to something else like `shouldDropdownShow`, `shouldDropdownDisplay`, etc?

Reply:
`shouldDropdownDisplay` sounds better 👍